feature/signupflow/onboarding/src/main/java/com/epmedu/animeal/signup/onboarding/presentation/OnboardingScreen.kt

```diff
@@ -52,7 +63,7 @@ private fun OnState(
             }
             is AuthenticationType.Facebook -> {
                 if (it.isPhoneNumberVerified) {
-                    navigator.navigateToTabs()
```

Comment:
Why we need those changes?

Reply:
Part of other approach I was trying, removed